src/comment.ts

```diff
@@ -10,7 +10,7 @@ export type CreateCommentWithTargetInput = {
   sha: string;
   regBranch: string;
   artifactName: string;
-  targetRun: Run;
+  targetHash: string | null;
```

Comment:
Why did you edit this part? This change is unnecessary for this feature, and I would appreciate it if you could keep changes to a minimum.

Reply:
Reverted, marking the comment as resolved 👍